clients/cloud/kafkacat/README.md

```diff
@@ -0,0 +1,77 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using [kafkacat](https://github.com/edenhill/kafkacat).
+
+
+# Prerequisites
+
+* [kafkacat](https://github.com/edenhill/kafkacat) installed on your machine.  You must [build](https://github.com/edenhill/kafkacat#build) `kafkacat` from the latest master branch to get the `-F` functionality that makes it easy to pass in the configuration to your Confluent Cloud configuration file.
```

Comment:
It would be good if we could get the new kafkacat release out first so people can brew it.
What's the release timeline for these examples?

Reply:
Target is March.  When's the release for the new `kafkacat`?